### DISCUSSION TITLE
Building docker image on ARM raspberryPi

### DISCUSSION THREAD

**Discussion by jamroks** [loxilb-io/loxilb#578] [General]
Hello i'm trying to build a docker image of loxilb on my raspebrrypi, but the build failed with the following error, can you please advise  or give me instruction i can follow to successfully build and run on raspberryPi 4 8G . please ? thx 

********************************************************************
315.8   INSTALL  bpf.h libbpf.h btf.h libbpf_common.h libbpf_legacy.h xsk.h bpf_helpers.h bpf_helper_defs.h bpf_tracing.h bpf_endian.h bpf_core_read.h skel_internal.h libbpf_version.h usdt.bpf.h
315.8   INSTALL  build/libbpf.pc
315.8   INSTALL  build/libbpf.a build/libbpf.so build/libbpf.so.0 build/libbpf.so.0.8.1
315.9 /iproute2-main
315.9 TC schedulers
315.9  ATM	no
315.9
315.9 libc has setns: yes
316.1 SELinux support: no
316.1 libbpf support: no
316.1 	libbpf version 0.8.1 is too low, please update it to at least 0.1.0
316.1 	LIBBPF_FORCE=on set, but couldn't find a usable libbpf

********************************************************************

I clone the repository from the 'main' branch  , just so you know

**Comment by jamroks** [comment]
i think it's this error that make the build failed 

***************************************************************
libbpf version 0.8.1 is too low, please update it to at least 0.1.0
LIBBPF_FORCE=on set, but couldn't find a usable libbpf
***************************************************************

**Reply by TrekkieCoder** [reply]
Recent changes broke arm build. Fixed some stuff. Can you retry again (I assume you are trying the following in the Raspberry Pi board")  -
```
docker build --no-cache -t ghcr.io/loxilb-io/loxilb:latest .
```

If it is not working, can you please tell the output of - 
```
uname -m
```

**Comment by UltraInstinct14** [comment]
You can simply used the latest arm-docker 
```
ghcr.io/loxilb-io/loxilb:latest-arm64 
```

**Comment by TrekkieCoder** [comment]
The feature [request](https://github.com/loxilb-io/loxilb/issues/516) for multi-arch docker is complete. You can simply use the same image ```ghcr.io/loxilb-io/loxilb:latest``` for amd64/arm64.

**Reply by jamroks** [reply]
Thank you so much, i can resume my experiment /  exploration and discovery of loxilb in raspberrypi  board environment for the edge location ...i will keep you posted asap how that goes
